src/scribe_data/extract_transform/languages/Russian/translations/translate_words.py

```diff
@@ -0,0 +1,32 @@
+import json
+import os
+import signal
+import sys
+
+PATH_TO_SCRIBE_ORG = os.path.dirname(sys.path[0]).split("Scribe-Data")[0]
+PATH_TO_SCRIBE_DATA_SRC = f"{PATH_TO_SCRIBE_ORG}Scribe-Data/src"
+sys.path.insert(0, PATH_TO_SCRIBE_DATA_SRC)
+
+from scribe_data.utils import translate_to_other_languages, translation_interrupt_handler
+
+translate_script_dir = os.path.dirname(os.path.abspath(__file__))
+words_to_translate_path = os.path.join(translate_script_dir, 'words_to_translate.json')
+
+with open(words_to_translate_path, 'r', encoding='utf-8') as file:
+    json_data = json.load(file)
+
+word_list = {}
+for item in json_data:
+    word_list.append(item["word"])
+
+src_lang="Russian"
```

Comment:
Modify the `src_lang` variable to specify the source language for the translations. 
Keep the rest of the `translate_words.py` script unchanged.

Reply:
🔥🔥🔥 Nice, @shashank-iitbhu :) I'll take a bit more of a look at this, and maybe we can move this to an all caps var at the top. I can handle this myself though 😊